schema/deploy/tasks.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy todo_app:tasks to pg
+-- requires: appschema
+
+BEGIN;
+
+CREATE TABLE todo_app.tasks (
+    id  bigint unique,
+    task varchar(50),
+    completed bool,
+    date_created date default current_date,
```

Comment:
`date` won't work well with time zones. `current_date` will give you the date in you database's time zone, which may or may not be the same time zone as a user.
`timestamptz` is preferred to store anything time-related. We have some handy function you can reuse in our projects to create those `created/updated/archived` columns